libgearman-server/plugins/queue/sqlite/instance.cc

```diff
@@ -593,6 +598,17 @@ gearmand_error_t Instance::replay_loop(gearman_server_st *server)
                                "failed to reset REPLAY prep statement: %s", sqlite3_errmsg(_db));
   }
 
+  /* truncate database now if in store_on_shutdown mode */
+  if (is_store_on_shutdown())
+  {
+    std::string query("DELETE FROM ");
+    query+= _table;
+    if (_sqlite_dispatch(query) == false) {
+      return gearmand_log_gerror(GEARMAN_DEFAULT_LOG_PARAM, GEARMAND_QUEUE_ERROR,
+                                "failed to truncate table: %s", sqlite3_errmsg(_db));
```

Comment:
Yet another super-minor nitpick: I think you need one more space before `"failed to truncate table: %s"`. The first double quote should be directly under the "G" in `GEARMAN_DEFAULT_LOG_PARAM`.

Otherwise, based on @SpamapS comments in issue #369, I think we're good to go here! The only issue before we can merge is that our CI needs to be updated (see issue #366), but I'll try to fix that soon (if nobody beats me to it). You'll probably need to rebase after that's resolved so that the CI can run.

Reply:
Our CI is working again. @sni, please add that space and rebase your branch. Thank you!

Reply:
great, i rebased this pr and added the space.